test/suites/storage_driver_ceph.sh

```diff
@@ -130,6 +130,22 @@ test_storage_driver_ceph() {
     lxc storage delete "lxdtest-$(basename "${LXD_DIR}")-pool1"
     lxc storage delete "lxdtest-$(basename "${LXD_DIR}")-pool2"
     ceph --cluster "${LXD_CEPH_CLUSTER}" osd pool rm "lxdtest-$(basename "${LXD_DIR}")-existing-osd-pool" "lxdtest-$(basename "${LXD_DIR}")-existing-osd-pool"  --yes-i-really-really-mean-it
+
+
+    # Test that the pool size setting only applies to the created pool, and not any other pools.
+    pool1="$("lxdtest-$(basename "${LXD_DIR}")-pool1")"
+    pool2="$("lxdtest-$(basename "${LXD_DIR}")-pool2")"
+    lxc storage create "${pool1}" ceph volume.size=25MiB ceph.osd.pg_num=16 ceph.osd.pool_size=1
+    ceph --cluster "${LXD_CEPH_CLUSTER}" osd pool get "${pool1}" size --format json | jq '.size' | grep -q 1
```

Comment:
Using shell comparison should help with debugging as we will see what `.size` we got on the left hand side of the comparison:
```suggestion
    [[ "$(ceph --cluster "${LXD_CEPH_CLUSTER}" osd pool get "${pool1}" size --format json | jq '.size')" = "1" ]]
```

Reply:
Fixed